src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -34,7 +34,11 @@ public CommandResult execute(Model model) {
                 String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
     }
 
-    @Override
```

Comment:
Perhaps this could be added as a separate PR, since this PR is related to updating the DG? 😸